agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -247,7 +242,15 @@ private static void logConfigurationWarnings(Configuration config) {
         config.sampling.requestsPerSecond = config.sampling.limitPerSecond;
       }
     }
-
+    if (config.preview.authentication.enabled && !config.authentication.enabled) {
+      configurationLogger.warn(
+          "\"authentication\" is no longer in preview and it has been GA since 3.4.18");
+      config.authentication = config.preview.authentication;
+    }
+    if (config.authentication.clientSecret != null) {
+      configurationLogger.warn(
+          "\"CLIENTSECRET\" typed of AAD authentication has been deprecated since 3.5.0 GA. Please use \"user-assigned or system-assigned identify\" instead.");
```

Comment:
also, why caps `CLIENTSECRET`?

```suggestion
          "\"CLIENTSECRET\" typed of AAD authentication has been deprecated since 3.5.0 GA. Please use \"user-assigned\" or \"system-assigned identify\" instead.");
```